README.md

```diff
@@ -23,6 +23,14 @@
    rosdep install --from-paths . --ignore-src -y
    ```
 
+1. Install our non-ROS dependencies
+
+   ```bash
+   # In the ateam_ws directory
+   source /opt/ros/galactic/setup.bash
+   ./ateam_ui/install_deps.sh
```

Comment:
Unless we have our workspaces set up differently the path for me was `./src/software/ateam_ui/install_deps.sh`. Also, when I tried to run it on a fresh installation it just says "Upgrading node to latest stable" every time and then doesn't actually install any of the packages other than npm.